packages/plugins/test/Metatx.spec.ts

```diff
@@ -0,0 +1,115 @@
+import * as path from 'path'
+
+import hre from 'hardhat'
+import { Contract } from 'ethers'
+import {
+  chugsplashClaimAbstractTask,
+  chugsplashProposeAbstractTask,
+  getChugSplashManager,
+  readUnvalidatedChugSplashConfig,
+  readValidatedChugSplashConfig,
+} from '@chugsplash/core'
+import { FORWARDER_ADDRESS, ForwarderArtifact } from '@chugsplash/contracts'
+import { expect } from 'chai'
+
+import { getArtifactPaths } from '../../plugins/dist'
+import { createChugSplashRuntime } from '../../plugins/src/utils'
+
+const configPath = './chugsplash/Metatx.config.ts'
+
+describe('Meta txs', () => {
+  process.env['LOCAL_TEST_METATX_PROPOSE'] = 'true'
```

Comment:
We only perform meta transactions on live networks when the API key is available. I've included this environment variable to override the network check for testing.